components/prism-powershell.js

```diff
@@ -9,26 +9,38 @@ Prism.languages.powershell = {
 			lookbehind: true
```

Comment:
Here, we might want to get rid of the last group and just end the regexp with `#.+`, unless you really need to consume the line feed.